doc/dev/CODING_STYLE.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: default
+title: Coding Style
+parent: Developer Documentation
+nav_order: 4
+---
+
+# F3D Coding Style
+
+F3D use different coding styles in each component, however there are some common rules
+
+## Common rules
+
+Overall syntax:
+ - CamelCase.
+ - Avoid using `using`.
```

Comment:
`using` is fine, but `using namespace` (particularly in headers) should be avoided.